packages/components/organisms/f-header/README.md

```diff
@@ -80,7 +80,7 @@ The props that can be defined are as follows:
 | locale                    | `String`      | `en-GB` | Sets the locale of the component (which determines what theme and translations to use.<br><br>If the application consuming the `f-header` component is using the vue `i18n` module, then the locale from that module will be used when this prop isn't defined. When this prop is defined, it takes precedence over the locale defined by the `i18n` module.<br><br>If not defined and the `i18n` module isn't present, the default locale used is `en-GB`.|
 | customNavLinks            | `Array`       | `[]`    | Array containing objects representing the custom links. To contain `text`, `url` and `gtm` properties. |
 | errorLog                  | `Function`    | `-`     | Function passed in for logging errors with the `fetchUserInfo` method. |
-| headerBackgroundTheme     | `String`      | `white` | Sets the background theme for the header component.<br><br>When set to `white` the header will be white with the default logo colour.<br>When set to `transparent` the header will be transparent with a white logo.<br>When set to `highlight` the header will use the primary brand colour as the background colour with a white logo.<br>When set to `disappearingWhite` the header will be transparent with with the default logo colour on wide screens, and white with box-shadow at the bottom of the header on narrow screens.  |
+| headerBackgroundTheme     | `String`      | `white` | Sets the background theme for the header component.<br><br>When set to `white` the header will be white with the default logo colour.<br>When set to `transparent` the header will be transparent with a white logo.<br>When set to `highlight` the header will use the primary brand colour as the background colour with a white logo.<br>When set to `disappearingWhite` the header will be white with no box-shadow at the bottom and with the default logo colour on wide screens, and white with box-shadow at the bottom of the header on narrow screens.  |
```

Comment:
Is the name `disappearingWhite` still appropriate if it's no longer transparent on wide screens?

Reply:
good question! Maybe "plainWhite"? The only difference right now is that there's no `box-shadow` at the bottom. @ashleynolan what do you think?

Reply:
I'd probably rename it – maybe something like `seamlessWhite`, as it's intended to merge the header straight into the page content?

Reply:
The alternative would be just to make it a separate prop which controls whether there should be a `boxShadow` or not, which may be more appropriate?

Reply:
it happens only on wide screens. There's a `boxShadow` on narrow screens.

Reply:
Ok, maybe keep it as a theme then in that case and just call it `whiteSeamless`